code/datums/abilities/wraith.dm

```diff
@@ -207,7 +208,7 @@
 	targeted = 1
 	target_anything = 1
 	pointCost = 1000
-	cooldown = 5000 //5 minutes
+	cooldown = 5000
```

Comment:
bruh at least make it use the defines instead of just removing the comment, jeez
also this isn't 5 minutes, but 8.3......

Reply:
The only bad math allowed in the code I modify should be done by me!